cross/ser2net/Makefile

```diff
@@ -0,0 +1,28 @@
+PKG_NAME = ser2net
+PKG_VERS = 4.0
+PKG_EXT = tar.gz
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+PKG_DIST_SITE =  http://downloads.sourceforge.net/project/$(PKG_NAME)/$(PKG_NAME)
+PKG_DIST_NAME = $(PKG_DIR).$(PKG_EXT)
+
```

Comment:
for the sake of the order we prefere to define first the PKG_DIST_NAME than the PKG_DIST_SITE and both before PKG_DIR.
this will result in:
```
PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
PKG_DIST_SITE = http://downloads.sourceforge.net/project/$(PKG_NAME)/$(PKG_NAME)
PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
```

same for cross/gensio/Makefile